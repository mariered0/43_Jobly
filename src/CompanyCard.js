import React from "react";

import {Card, CardTitle, CardText} from "reactstrap"

const CompanyCard = () => {

    

    return (
        <div className="container">
            <Card body className="my-2" style={{width: '100%'}}>
                <CardTitle tag="h5">
                    {}
                </CardTitle>
                <CardText>
                    abc
                </CardText>
            </Card>
        </div>
    )
};

export default CompanyCard;