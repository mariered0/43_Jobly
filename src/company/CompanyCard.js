import React from "react";
import { useHistory } from "react-router-dom";

import { Card, CardTitle, CardText } from "reactstrap";

const CompanyCard = ({ name, desc, handle }) => {
    const history = useHistory();

    const handleClick = () => {
        const path = `/companies/${handle}`;
        history.push(path);
    }

  return (
    <div className="container">
      <Card
        body
        className="my-2"
        style={{ width: "100%", cursor: "pointer" }}
        onClick={handleClick}
        >
        <CardTitle tag="h5">{name}</CardTitle>
        <CardText>{desc}</CardText>
      </Card>
    </div>
  );
};

export default CompanyCard;
