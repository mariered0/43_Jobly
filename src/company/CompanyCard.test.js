import React from "react";
import { render } from "@testing-library/react";
import CompanyCard from "./CompanyCard";

it("renders without crashing", () => {
    render(<CompanyCard />);
});

it("matches snapshot", () => {
	const { asFragment } = render(<CompanyCard />);
	expect(asFragment()).toMatchSnapshot();
});