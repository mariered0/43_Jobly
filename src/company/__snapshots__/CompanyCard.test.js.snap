// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="my-2 card card-body"
      style="width: 100%; cursor: pointer;"
    >
      <h5
        class="card-title"
      />
      <p
        class="card-text"
      />
    </div>
  </div>
</DocumentFragment>
`;
